src/memex/search/client.py

```diff
@@ -15,13 +18,44 @@ class Client(object):
     :param index: index name
     """
 
-    class t(object):
+    class t(object):  # noqa
         """Document types"""
         annotation = 'annotation'
         document = 'document'
 
     def __init__(self, host, index, **kwargs):
         self.index = index
-        self.conn = elasticsearch.Elasticsearch([host],
-                                                verify_certs=True,
-                                                **kwargs)
+        self.conn = Elasticsearch([host], verify_certs=True, **kwargs)
+
+    def get_aliased_index(self):
+        """
+        Fetch the name of the underlying index.
+
+        Returns ``None`` if the index is concrete.
+        """
+        try:
+            result = self.conn.indices.get_alias(name=self.index)
+        except NotFoundError:  # self.index is a concrete index
```

Comment:
The docstring says that it returns `None` if the index is concrete, but it will also return `None` when the alias or index does not exist at all. Same thing with the code comment on this line.